src/python/pants/engine/process_test.py

```diff
@@ -110,116 +108,6 @@ def cat_stdout_rules():
     return [cat_files_process_result_concatted, QueryRule(Concatted, (CatExecutionRequest,))]
 
 
-@dataclass(frozen=True)
-class JavacVersionExecutionRequest:
```

Comment:
These Javac tests were useful when we had barely any v2 code, but now they're not pulling their weight. They are costly because they require having Java installed in the test environment.

We can add Java tests when we add Java as a proper language.